src/ui/dashboard/sidenav.tsx

```diff
@@ -14,37 +14,31 @@ export default async function SideNav({
   presentations: Presentation[];
 }) {
   return (
-    <div className="flex h-full w-full flex-col space-y-2 bg-gray-300 p-2 pt-4">
-      <Link className="flex rounded-md bg-black px-4 py-6" href="/dashboard">
-        <div className="w-full text-white">
-          <AcmeLogo>
-            <p className="text-2xl">Simonyi Konferencia</p>
-          </AcmeLogo>
-        </div>
-      </Link>
+    <div className="flex h-full w-[400px] flex-col items-center overflow-hidden bg-slate-200 shadow-inner shadow-slate-500/10">
+      <div className="flex w-full items-center justify-center rounded-br-lg bg-white p-5 shadow-md shadow-slate-500/10">
+        <Link href="/dashboard">Simonyi Konferencia</Link>
```

Comment:
This lacks visual cues of a primary pressable element.
Lack of clear distinction between this and the scrollable list of links bellow it.